.github/workflows/progress-tracker.yml

```
@@ -0,0 +1,80 @@
+name: Progress Tracker
+
+on:
+  workflow_dispatch:
+  issues:
+    types:
+      - edited
+
+jobs:
+  track-progress:
```

If we are going to run it on every edit (see above), then we should at least bomb out early:

```suggestion
  track-progress:
      if: contains(github.event.issue.labels.*.name, '2020 chapter')
```

Annoyingly can't seem to use environment variables in the `contains` function so have to hard code this. Should make a comment above to note if changing FILTER_LABEL then need to change here too.

---

Since I have decided to not go with triggering on issue edit, I think we can safely pass this for now. Though, I have limited the workflow to this repo only.

---

Good idea on limiting to this repo.

Depending what way we go on editing versus cron it we may still want to add this restriction so leaving this unresolved for now.

---

It would have been better if we could skip it on the job level instead of the step level, but I think we will go with the less brittle approach here to not specify labels in two different places and skip the step instead. This will be a little more work for GH, but that's okay.